app/services/search/criteria_inventor.rb

```diff
@@ -30,6 +31,14 @@ def keyword
     get_keywords_from_job_title.presence unless @vacancy.job_roles.present?
   end
 
+  def set_location
+    @location ||= if @vacancy.organisations.many?
```

Comment:
Will that postcode then show up in the job alert? That might look a bit weird if the user never entered the mean postcode.

Reply:
Yes. Not sure what we should show up here - I began with using just lat and lng everywhere, but only afterwards did I realise that we'd need something human-readable when a job alert is created from the search!